addons/sale/tests/test_accrued_sale_orders.py

```diff
@@ -137,3 +137,35 @@ def test_analytic_account_accrued_order(self):
             {'account_id': self.account_expense.id, 'debit': 12000.0, 'credit': 0.0, 'analytic_distribution': {str(self.analytic_account_a.id): 66.67, str(self.analytic_account_b.id): 33.33, str(self.analytic_account_c.id): 100.0}},
 
         ])
+
+    def test_product_name_in_accured_revenue_entry(self):
+        self.sale_order.order_line.qty_delivered = 5
+
+        so_context = {
+            'active_model': 'sale.order',
+            'active_ids': [self.sale_order.id],
```

Comment:
Another way of doing this
```suggestion
            'active_ids': self.sale_order.ids,
```